service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -365,6 +365,8 @@ public List<ControlledResource> listControlledResources(
       params.addValue("cloud_platform", cloudPlatform.toSql());
     }
 
+    sql += " ORDER BY created_date desc";
```

Comment:
It would be good to add a comment to the code about why reverse order is being returned.